examples/webgl_geometry_tube.html

```diff
@@ -0,0 +1,289 @@
+        <script src="js/controls/DragControls.js"></script>
+
+        <script src='js/libs/dat.gui.min.js'></script>
+        <script src="js/libs/stats.min.js"></script>
+        
+        <div id="info">
+            <a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - geometry tube<br>
+            <label style="color: yellow">Drag points to modify the tube.</label>
+        </div>
+
+		<script>
+
+            var camera, scene, renderer, controls, mesh, dragControls;
+            var pointGroup = new THREE.Group();
+            var size = 128;
+
+            var parameters = {
+                path: new THREE.CatmullRomCurve3( getInitialPoints() ),
+                curve: 'CatmullRomCurve3',
+                side: 'FrontSide',
+                tubularSegments: 64,
+                radius: 1,
+                radialSegments: 8,
+                closed: false,
+                divisions: 5,
+                opacity: 0.5
+            };
+
+            init();
+            animate();
+
+            function init() {
+
+                renderer = new THREE.WebGLRenderer();
+                renderer.setPixelRatio( window.devicePixelRatio );
+                renderer.setSize( window.innerWidth, window.innerHeight );
+                document.body.appendChild( renderer.domElement );
+
+                scene = new THREE.Scene();
+                scene.background = new THREE.Color( 0x222222 );
+
+                scene.add( pointGroup );
+
+                camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 1, 1000 );
+                camera.position.set( 0, 0, 10 );
+
+                controls = new THREE.TrackballControls( camera, renderer.domElement );
+                controls.minDistance = 200;
+                controls.maxDistance = 500;
+
+                scene.add( new THREE.AmbientLight( 0x222222 ) );
+
+                var light = new THREE.PointLight( 0xffffff );
+                light.position.copy( camera.position );
+                scene.add( light );
+
+                var gui = new dat.GUI( { width: 300 } );
+                gui.add( parameters, 'tubularSegments', 3, 128, 1 ).name( 'Tubular Segments' ).onChange( updateObject );
+                gui.add( parameters, 'radius', 0.1, 16, 0.01 ).name( 'Radius' ).onChange( updateObject );
+                gui.add( parameters, 'radialSegments', 3, 16, 1 ).name( 'Radial Segments' ).onChange( updateObject );
+                gui.add( parameters, 'closed' ).name( 'Closed' ).onChange( updateObject );
+
+                gui.add( parameters, 'divisions', 2, 16, 1 ).name( "Divisions" ).onChange( setPoints );
+                gui.add( parameters, 'opacity', 0, 1, 0.01 ).name( "Opacity" ).onChange( updateOpacity );
+
+                gui.add( parameters, 'curve', [ 'CatmullRomCurve3', 'CubicBezierCurve3', 'LineCurve3', 'QuadraticBezierCurve3' ] ).name( 'Curve' ).onChange( updateObject );
```

Comment:
If I select `LineCurve3` and then `CatmullRomCurve3` again, I only see two control points and not the original ones. Is this intended?

Reply:
I wasn't aware there was already an example for Tube Geometry. I only checked examples page. So now that this example doesn't have a purpose, I'll just close this pull request.